ts.js

```diff
@@ -11,7 +11,8 @@ module.exports = {
     'standard-with-typescript'
   ],
   plugins: [
-    'etc'
+    'etc',
+    'import'
```

Comment:
I think this section may be unnecessary as we extend the config from `js.js`?  At least running locally I can format `.ts` imports with the `.js` rules.

```suggestion
    'etc'
```